modules/models/src/main/java/org/eclipse/xpanse/modules/models/service/deploy/exceptions/EulaNotAccepted.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+package org.eclipse.xpanse.modules.models.service.deploy.exceptions;
+
+/**
+ * Exception thrown when deployment service does not accept Eula.
```

Comment:
Exception thrown when the service template defines an EULA but the end user has requested for the service without accepting it.